cluster/gce/gci/configure-helper.sh

```diff
@@ -83,18 +148,124 @@ function safe-format-and-mount() {
   mount -o discard,defaults "${device}" "${mountpoint}"
 }
 
-# Local ssds, if present, are mounted at /mnt/disks/ssdN.
+# Creates a bind mounting from the /mnt/disks/ fs mounted folder to
+# a new mountpoint in /mnt/disks/by-uuid/ using the devices UUID 
+function unique-uuid-bind-mount(){
+  local device=$1
+  local interface=$2
+  local mountpoint=$3
+
+  # Trigger udev refresh so that newly formatted devices are propagated in by-uuid
+  udevadm control --reload-rules
+  udevadm trigger
+  udevadm settle
+
+  # Get the real UUID of the device
+  if [[ ${interface} == "nvme" ]]; then
```

Comment:
Instead of doing this check here, could you do it outside, and then pass in the actual device path in this function?

Reply:
Can we consolidate the scsi vs nvme checks to one place?